tests/ui/empty/empty-macro-use.stderr

```diff
@@ -4,7 +4,8 @@ error: cannot find macro `macro_two` in this scope
 LL |     macro_two!();
    |     ^^^^^^^^^
    |
-help: consider importing this macro
+   = help: consider importing this macro:
+           use two_macros::macro_two;
    |
 LL + use two_macros::macro_two;
    |
```

Comment:
Not quite sure how this happened given the logic change, but I would like us to avoid changing the message for the structured suggestion case.